src/PostageStamp.sol

```diff
@@ -61,10 +61,14 @@ contract PostageStamp is AccessControl, Pausable {
         address owner;
         // Current depth of this batch.
         uint8 depth;
+        //
+        uint8 bucketDepth;
```

Comment:
i starting to doubt this was a good name suggests the log size of a bucket. eitger call it 
- `maxNeighbourhoodDepth` OR
- `minCollisionDepth`